docs/modules/ROOT/pages/push.adoc

```diff
@@ -82,6 +83,11 @@ pushObject.on('registration').subscribe(data => {
     // Error on device registration
   });
 });
+
+// Notification handler
+PushService.pushObject.on('notification').subscribe(notification => {
```

Comment:
should be: `pushObject.on` (without `PushService`, that is showcase specific).

Reply:
Awesome catch! We may point people to showcase application on review. This will reduce number of issues in docs code snippets.